packages/backup-restore-operator/charts/templates/rancher-resourceset.yaml

```diff
@@ -11,6 +11,10 @@ resourceSelectors:
   - apiVersion: "v1"
     kindsRegexp: "^Secret$|^serviceaccounts$"
     namespaceRegexp: "^cattle-|^p-|^c-|^local$|^user-|^u-"
+  - apiVersion: "v1"
+    kindsRegexp: "^configmaps$"
+    namespaces:
+      - "cattle-system"
```

Comment:
This would grab all configmaps in cattle-system ? We want to do that right.

Reply:
@cbron yes looks like it. We already backup secrets and serviceaccounts from this namespace too. Plus all index info for clusterrepos will be in these configmaps